packages/fxa-content-server/tests/functional.js

```diff
@@ -22,7 +22,6 @@ module.exports = [
   'tests/functional/cookies_disabled.js',
   'tests/functional/delete_account.js',
   'tests/functional/email_opt_in.js',
-  'tests/functional/email_service.js',
```

Comment:
@shane-tomlinson did you want to delete this? this is unrelated to email_first ?

Reply:
It is unrelated, but we are using the email-service everywhere now so it seems like a bit of a superfluous test.

Reply:
Brought the test back.